nipype/interfaces/niftyreg/regutils.py

```diff
@@ -295,6 +295,15 @@ class RegToolsInputSpec(NiftyRegCommandInputSpec):
         desc=desc,
         argstr='-smoG %f %f %f')
 
+    # Interpolation type
+    inter_val = traits.Enum(
+        'NN',
+        'LIN',
+        'CUB',
+        'SINC',
+        desc='Interpolation order to use to warp the floating image',
+        argstr='-interp %d')
```

Comment:
you'll want to change this to `%s`

Reply:
Thanks for the feedback. I've checked the source code and this parameter (of the command line tool) expects `0`, `1`, `3` or `4`, as in `RegResample`:
https://github.com/nipy/nipype/blob/045b28ef9056fac1107bc4f0707859d043f3bfd1/nipype/interfaces/niftyreg/regutils.py#L61-L68

So I basically copied that section from `RegResample`. Should anything be modified for both interfaces? I don't know much about traits, but it feels like the current code can only produce `0`, `1`, `2` or `3`.

Also, I'm a bit new to PRs. To change something, should I just commit in my local fork and push to the same branch normally?

Reply:
You'll need to copy this part as well:

https://github.com/nipy/nipype/blob/045b28ef9056fac1107bc4f0707859d043f3bfd1/nipype/interfaces/niftyreg/regutils.py#L123-L128

Reply:
I see. Although shouldn't line 124 be
`inter_val = {'NN': 0, 'LIN': 1, 'CUB': 3, 'SINC': 4}`
instead of
`inter_val = {'NN': 0, 'LIN': 1, 'CUB': 3, 'SINC': 5}`
?

Reply:
I'm not sure why 2 and 4 are skipped, perhaps @mmodat has some input.

```python
# If change is needed, it should look something like this
inter_val = {'NN': 0, 'LIN': 1, 'CUB': 2, 'SINC': 3}
```

Reply:
Looks like it. https://cmiclab.cs.ucl.ac.uk/mmodat/niftyreg/blob/master/reg-apps/reg_resample.cpp#L65

Their documentation could certainly be clearer on the point.